setools/policyrep/mlsrule.pxi

```diff
@@ -67,7 +67,10 @@ cdef class MLSRule(PolicyRule):
             yield self
 
     def statement(self):
-        return f"{self.ruletype} {self.source} {self.target}:{self.tclass} {self.default};"
+        if self.policy.gen_cil:
+            return f"(rangetransition {self.source} {self.target} {self.tclass} {self.default})"
+        else:
+            return f"{self.ruletype} {self.source} {self.target}:{self.tclass} {self.default};"
```

Comment:
The other places in the change should have separate methods for each policy lang, unless they are simple one-liners like this.